### DISCUSSION TITLE
Ethernet port speed switch

### DISCUSSION THREAD

**Discussion by olegin15** [espressif/arduino-esp32#7855] [Q&A]
Hi.

I need to set the LAN8710 ethernet port speed to 10Mbps. 
I know that this feature only appeared in ESP-IDF 5.0. 
Is it possible in the current version of Arduino to switch the speed of the ethernet port?

Thanks.

**Comment by Ing-Dom** [comment]
really 5.0 ?

I found this in 4.4
 https://docs.espressif.com/projects/esp-idf/en/release-v4.4/esp32/api-reference/network/esp_eth.html#_CPPv4N13esp_eth_mac_s9set_speedE
but struggling to call it from Arduino because I dont know how to get the phy object...
